wiki/mapping/map-format.md

```diff
@@ -313,16 +313,456 @@ The exact specifics of what goes in `_customData` is entirely dependent on commu
+The bottommost layer is located at layer 0, and inceases to the topmost layer located at index 2.
+
+#### tc
+An integer number which represents the tail direction of the arc.
+
+|`_d`|Result|
+|:-------------------:|-------------------|
+|`0`|Up|
+|`1`|Down|
+|`2`|Left|
+|`3`|Right|
+|`4`|Up Left|
+|`5`|Up Right|
+|`6`|Down Left|
+|`7`|Down Right|
+|`8`|~~Any (Dot Note).~~ Functions as Down.|
+
+#### tmu
+A float which represents how far the arc goes from the tail of the arc.
+
+#### m
+An integer number which represents how the arc curves from the head to the mid point of the arc.
+
+|`m`|Result|
+|:-------------------:|-------------------|
+|`0`|Straight|
+|`1`|Clockwise|
+|`2`|Counter-Clockwise|
+
+### Burst Sliders
+These describe chain and links. If the head of a chain matches a note's time and position,
+then the chain will connect with the note and alter how the note is scored.
+
+```json
+{
+  "b": 10.0,  // Beat
+  "x": 1,     // Head x
+  "y": 0,     // Head y
+  "c": 0,     // Color
+  "d": 1,     // Head direction
+  "tb": 15.0, // Tail Beat
+  "tx": 2,    // Tail x
+  "ty": 2,    // Tail y
+  "sc": 3,    // Segment count
+  "s": 0.5    // Squish factor
+}
+```
+
+#### b
+The time, in beats, where this head of this object reaches the player.
+
+#### c
+An integer which represents the color of the note.
+|`_c`|Result|
+|:---:|-----|
+|`0`|Red|
+|`1`|Blue|
+
+#### x
+An integer number, from 0 to 3, which represents the column where the head of the arc is located.
+The far left column is located at index 0, and increases to the far right column located at index 3.
+
+#### y
+An integer number, from 0 to 2, which represents the layer where the head of the arc is located.
+The bottommost layer is located at layer 0, and inceases to the topmost layer located at index 2.
+
+#### d
+An integer number which represents the head direction of the arc.
+
+|`_d`|Result|
+|:-------------------:|-------------------|
+|`0`|Up|
+|`1`|Down|
+|`2`|Left|
+|`3`|Right|
+|`4`|Up Left|
+|`5`|Up Right|
+|`6`|Down Left|
+|`7`|Down Right|
+|`8`|Any (Dot Note)|
+
+#### tb
+The time, in beats, where this tail of this object reaches the player.
+
+#### tx
+An integer number, from 0 to 3, which represents the column where the tail of the arc is located.
+The far left column is located at index 0, and increases to the far right column located at index 3.
+
+#### ty
+An integer number, from 0 to 2, which represents the layer where the tail of the arc is located.
+The bottommost layer is located at layer 0, and inceases to the topmost layer located at index 2.
+
+#### sc
+An integer number, greater than 0, which represents the number of segments in the burst slider.
+The head counts as a segment.
+
+#### s
+A float which represents squish factor. This is percentage of how much of the burst slider is contained in the path
+from `(x,y)` to `(tx, ty)`. High values can send the slider off the grid.
```

Comment:
It should be the other way around, how much of the path is used by the burst slider. Could also mention a little more clearly that this does not alter the shape of the path, and that values >1 will extend the path beyond the specified end point.